jekyll/_cci2/rerun-failed-tests-only.adoc

```diff
@@ -0,0 +1,275 @@
+
+```sh
+Installing circleci-tests-plugin-cli plugin.
+circleci-tests-plugin-cli plugin Installed. Version: 1.0.3349-470dac4
+DEBU[2023-04-10T22:36:53Z] Attempting to read from stdin. This will hang if no input is provided. 
+INFO[2023-04-10T22:36:53Z] starting batch execution                      batch_count=1 batches_processed=0 total_batches_for_job=3
+DEBU[2023-04-10T22:36:53Z] received test names: test_api.py
+```
+
+NOTE: The job should only re-run tests that are from a `classname` of `filename` that had at least one test failure when the "Re-run failed tests only" button is clicked. If you are seeing different behavior, comment on this https://discuss.circleci.com/[Discuss post] for support.
+
+[#additional-examples]
+== Additional examples
+
+[#configure-a-job-running-ruby-rspec-tests]
+=== Configure a job running Ruby (rspec) tests
+
+. Add the following gem to your Gemfile:
++
+```bash
+gem 'rspec_junit_formatter'
+```
+
+. Modify your test command to use `circleci tests run`:
++
+```yaml
+ - run: mkdir ~/rspec
+ - run:
+    command: |
+      circleci tests glob spec/**/*_spec.rb | circleci tests run --command="xargs bundle exec rspec --format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml" --verbose
+```
+
+. Update the `glob` command to match your use case. See the RSpec section in the xref:collect-test-data#rspec[Collect Test Data] document for details on how to output test results in an acceptable format for `rspec`.
+
+[#configure-a-job-running-ruby-cucumber-tests]
+=== Configure a job running Ruby (Cucumber) tests
+
+. Modify your test command to look something similar to:
++
+```yaml
+- run: mkdir -p ~/cucumber
+- run:
+    command: |
+    circleci tests glob features/**/*.feature | circleci tests run --command="xargs bundle exec cucumber --format junit --out ~/cucumber/junit.xml" --verbose
+```
+
+. Update the `glob` command to match your use case. See the Cucumber section in the xref:collect-test-data#cucumber[Collect Test Data] document for details on how to output test results in an acceptable format for `Cucumber`.
+
+[#configure-a-job-running-cypress-tests]
+=== Configure a job running Cypress tests
+
+. Install the `cypress-multi-reporters` and `mocha-junit-reporter` dependencies. If using `npm`, run the following on your local machine:
++
+```bash
+npm install --save-dev cypress-multi-reporters mocha-junit-reporter
+```
++
+Your `package.json` / `package-lock.json` will be updated and committed on your next push from that branch. 
+
+. Create and set up the reporter config file, if it does not already exist. In this example, the file is named `reporter-config.json`.
++
+```json
+{
+  "reporterEnabled": "spec, mocha-junit-reporter", // set the reporters
+  "reporterOptions": {
+    "mochaFile": "results/junit/junit-[hash].xml", // each suite produces its own junit :(, save them with unique hash
```

Comment:
Want to double check that the `:(` is supposed to be in the comment

Reply:
you can get rid of it.